### DISCUSSION TITLE
404 Error when trying to purchase more seats for own enterprise.

### DISCUSSION THREAD

**Discussion by BPPGitHubAdmin** [community/community#39910] [Enterprise]
### Select Topic Area

Bug

### Body

Whenever I try to purchase more seats for my own enterprise, I get a 404 msg and can't continue.

The URL hints that it's trying to go onto the billing inside settings. 

Has anyone encountered this bug before?

And is there a fix for it?

![Screen 1](https://user-images.githubusercontent.com/101800991/203724617-5a42c03c-a7e7-4647-a148-8a4aa22e5f63.png)

![Screen 2](https://user-images.githubusercontent.com/101800991/203724609-02ecd95a-cb8d-4fd6-bdce-256fa5dbfd56.png)

**Comment by MitkoVtori** [comment]
If you get 404 error message, this is most likely  github's problem. Anyways, you can try pasting your mouse on the link(without clicking it) and look down on your left what the url is. It mait tell you about the problem like, if github have problem with the current page. Let me know when the issue is gone :)
